content/docs/1.3.1/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -28,3 +41,22 @@ This command gives users the ability to restore a backup to a `raw` image or a `
         kubectl -n longhorn-system get pod restore-to-file -w
 
 After the pod status changed to `Completed`, you should able to find `<OUTPUT_FILE>` at e.g. `/tmp/restore` on the `<NODE_NAME>`.
+
+
+ ## Appendix
+
+ ### qcow2 vs. raw
+   || qcow2 | raw |
+   |---|---|---|
+   |Pros|<ul><li>Multiple snapshots</li><li>Smaller image, no sparse file</li><li>Optional Zlib compression</li><li>Optional AES encryption</li></ul>|<ul><li>Sparse file support</li><li>Portable</ii><li>better performance</li></ul>|
```

Comment:
Actually, qcow2 supports sparse feature in nature.
Compression and encryption are supported by qcow2. But if we didn't support them in the feature, we might not need to mention them here. Or, the description will mislead users.

I think we can emphasize when users can `qcow2` or when can use `raw`.